examples/tictactoe/src/components/AvailableGames.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+
+import { Game } from '../types/game';
+import GameCard from './GameCard';
+
+interface AvailableGamesProps {
+    games: Game[]
+}
+
+const AvailableGames: React.FC<AvailableGamesProps> = ({ games  }) => {
```

Comment:
you can also write the above like below to help with how verbose it can get and not have to deal with React changing it's types like it does sometimes. wouldn't actually change anything on this PR just adding a comment for learning

```
const AvailableGames = ({ games  }: AvailableGamesProps)
```

Reply:
ahh nice didn't know that